gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Fixed the order of Genres

### Description

Fixed the order of Genres populated into the model. Right now the Genres are populated in the decreasing order, made changes to populate them in increasing order. 

### Testing Directions



Follow the directions below to test it.

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py shell
from showup.models import Genre
Genre.objects.all()
```
You should see that genres are populated in increasing order. You can also verify this in 'Edit Profile'. You should see that the genres are now populated in increasing order.

### Checklist

- [yes ] I ran `git merge develop` before submitting this PR.
- [ yes] I understand that I may have to run `git merge develop` again after submitting this PR since new changes may have been pulled into `develop`.
- [yes ] I understand that it is MY responsibility to make sure that this PR does not have any conflicts.
